src/nrnpython/inithoc.cpp

```diff
@@ -43,7 +43,7 @@ extern char* nrnmpi_load(int is_python);
 #if NRNPYTHON_DYNAMICLOAD
 extern int nrnpy_site_problem;
 #endif
-#if !defined(NRNCMAKE) && NRNPYTHON_DYNAMICLOAD && !__MINGW32__
+#if NRNPYTHON_DYNAMICLOAD && !__MINGW32__
```

Comment:
this should be opposite? ie. `!defined(NRNCMAKE)` is false in case of `CMake` build and hence `if` block could be removed.

Reply:
That's what I also thought, however that made the CI jobs fail. I suspect `NRNCMAKE` wasn't really passed

Reply:
This is a puzzle. The purpose was that for windows the autotools build created c modules  like hoc36.dll via home grown scripts. But the cmake build creates proper names like ```hoc.cp36-win_amd64.pyd```  And that reminds me that nrn/src/mswin/nrncygso.sh.in is no longer needed.

Reply:
Ok will revert this to double check the build failure.

Reply:
sorry, this PR was a couple of weeks back, I remember now. 
So I removed all `NRNCMAKE` logic and references. It's normal this is not used anymore.

Reply:
> I suspect `NRNCMAKE` wasn't really passed

It was actually passed through `setup.py.in`

Reply:
@alexsavulescu : Ok. But if the condition was below:

```cpp
#if !defined(NRNCMAKE) && NRNPYTHON_DYNAMICLOAD && !__MINGW32__
   #define HOCMOD(a, b) HOCMOD_(a, b)
   #define HOCMOD_(a, b) a ## b
 #else
   #define HOCMOD(a, b) a
#endif
```

With `CMake` build system, we should have just left `else` part i.e. `#define HOCMOD(a, b) a` ? Or, Am I missing something?

Reply:
right. after some more testing it turns out the build passes just fine by keeping only the `else` part.